backend/sample.js

```diff
@@ -76,6 +91,9 @@ app.get("/", async (req, res) => {
   res.send(allFolders);
 });
 // folder page: get all notes
+//I use post methods here because it needs to get password for private folders 
+//and I can't think of an alternative. 
+//Please comment if you have any suggestions or alternative solutions!!!
```

Comment:
Ive been looking into a solution for the password for private folders and seem to be running into a similar issue